extensions/wikia/ImageReview/ImageReview.hooks.php

```diff
@@ -7,24 +7,24 @@ public static function setupHooks() {
 		\Hooks::register( 'WikiFactoryPublicStatusChange', [ $oImageReviewHooks, 'onWikiFactoryPublicStatusChange' ] );
 	}
 
-	public function onWikiFactoryPublicStatusChange( $city_public, $city_id, $reason ) {
+	public static function onWikiFactoryPublicStatusChange( $city_public, $city_id, $reason ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$reason". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)